vulture_os/services/rsyslogd/config/rsyslog_template_sentinel_ama.conf

```
@@ -0,0 +1,74 @@
+template(name="sentinel_ama" type="list" option.jsonf="on") {
+    constant(value="{")
```

Missing syslog header (remember this template is for syslog/file forwarding):

```suggestion
template(name="sentinel_ama" type="list" option.jsonf="on") {
    property(name="timegenerated") constant(value=" {{node.name}} ") property(name="$!tenants_name") constant(value="-darwin")
    constant(value="{")
```